sdk/monitor/monitor-opentelemetry-exporter/samples/@azure/storage-blob/README.md

```diff
@@ -0,0 +1,73 @@
+# Getting started with samples
+
+Code cloned from [azure-sdk-for-js/storage-blob/samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob/samples).
+
+## `@azure/monitor-opentelemetry-exporter`
+
+All changes required to enable exporting trace data to Azure Monitor were made in `tracing.ts` / `tracing.js`.
+
+In order for auto-instrumentation to take effect, `tracing` must be imported **before** you `import` or `require` any other module into your application!.
+
+![AzureMonitorImage](./images/storage.png)
+
+## Get account credentials of your storage account
+
+- In the [Azure Portal](https://portal.azure.com), go to **Dashboard > Storage > _your-storage-account_**.
+- Note down the "AccountName", "AccountKey" obtained at **Access keys** and "AccountSAS" from **Shared access signature** under **Settings** tab.
+  Before running any of the samples, update with the credentials you have noted down above.
+
+### Authenticating with Azure Active Directory
+
+If you have [registered an application](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) with an Azure Active Directory tenant, you can [assign it to an RBAC role](https://docs.microsoft.com/azure/storage/common/storage-auth-aad) in your Azure Storage account. This enables you to use the Azure.Identity library to authenticate with Azure Storage as shown in the [azureAdAuth.ts sample](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob/samples/typescript/src/azureAdAuth.ts).
```

Comment:
I don't think the `azureAdAuth.ts` sample exists in this tree, but I guess we're linking out to the real storage samples here - is that going to be confusing?

Reply:
Removed all of the dead links. For now I'm linking to the real storage samples due to the MD link checker, but once this is merged in I can switch over to monitor's copy of samples